js/web/lib/onnxjs/backends/webgl/ops/fuse-utils.ts

```diff
@@ -36,11 +36,10 @@ export function getActicationSnippet(attributes: InternalActivationAttributes) {
 }
 
 export const parseInternalActivationAttributes = (attributes: Attribute): InternalActivationAttributes => {
-  const activation = attributes.getString('__internal_activation', '');
+  const activation = attributes.getString('activation', '');
 
   if (activation === 'Clip') {
-    const clipMax = attributes.getFloat('__clip_max', 3.402823e+38);
-    const clipMin = attributes.getFloat('__clip_min', -3.402823e+38);
+    const [clipMin, clipMax] = attributes.getFloats('activation_params', [-3.402823e+38, 3.402823e+38]);
```

Comment:
I remember there was a change to move those into const CLIP_MIN and CLIP_MAX. we can reuse them

Reply:
good catch